src/linked_list/node.js

```diff
@@ -0,0 +1,28 @@
+export default class Node {
+  constructor( data ) {
+    let _data = data
+    let _nextNode = null
+
+    this._getData = function() {
+      return _data
+    }
+    this._getNextNode = function() {
+      return _nextNode
+    }
+    this._setNextNode = function( pointer ) {
+      _nextNode = pointer
+    }
```

Comment:
the functions below should be the ones getting the data from the nodes. you dont need these above methods.